musicbot/bot.py

```diff
@@ -1960,7 +1960,7 @@ def check(m):
         self.config.auto_playlist = not self.config.auto_playlist
         await self.safe_send_message(channel, "The autoplaylist is now " + ['disabled', 'enabled'][self.config.auto_playlist])
```

Comment:
Send this message to Discord *last*, while returning a Response object as previously mentioned.